include/wx/qt/radiobut.h

```diff
@@ -46,8 +46,7 @@ class WXDLLIMPEXP_CORE wxRadioButton : public wxControl
 
 private:
     QRadioButton *m_qtRadioButton;
-
-    static std::map<wxWindow*, QButtonGroup*> m_lastGroup;
+    QButtonGroup* m_qtButtonGroup;
```

Comment:
This still shouldn't be a member variable. You can use a local variable in `wxRadioButton::Create` and in `~wxRadioButton` there are already 2 other ways for accessing it.

Reply:
I will update to match this.